tests/tests_basics_files.yml

```diff
@@ -119,8 +128,12 @@
         __logging_file: "{{ __logging_system_log_dir }}/messages"
       include_tasks: tasks/test_logger.yml
 
-    - name: Check the files output config that the path is {{ __logging_system_log_dir }}/messages
-      command: /bin/grep '\*.info;mail.none;authpriv.none;cron.none.*{{ __logging_system_log_dir }}/messages' {{ __test_files_conf0 }}
+    - name: "Check the files output config that the path is
+      {{ __logging_system_log_dir }}/messages"
+      command: "/bin/grep
```

Comment:
with `command` it is better to use `>-` which will generate a single line string with each newline replaced with a single space